### ISSUE TITLE
Nothing happened when run. Fedora 34

### ISSUE THREAD

**Issue by bigkrp** [Baldomo/asusctltray#3]
try run after install, but nothing happened.

When run `/usr/local/bin/asusctltray` got messages:

```
/usr/local/bin/asusctltray:64: DeprecationWarning: Gtk.StatusIcon.set_visible is deprecated
  self.icon.set_visible(True)
/usr/local/bin/asusctltray:68: DeprecationWarning: Gtk.StatusIcon.set_from_pixbuf is deprecated
  self.icon.set_from_pixbuf(self.image_pixbuf)
```

**Comment by Baldomo** [comment]
The script works on every desktop I tried which supports Gtk StatusIcons. I cannot reproduce the issue, so I would like to have more details on your setup if possible.

**Comment by bigkrp** [comment]
```
uname -a
Linux localhost.localdomain 5.12.8-300.fc34.x86_64 #1 SMP Fri May 28 15:20:54 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
```

When i run via gui, just nothing happened

**Comment by Baldomo** [comment]
Could you please describe what desktop environment you're using?

**Comment by bigkrp** [comment]
Gnome 40.1.0, X11

**Comment by Baldomo** [comment]
I see. Apparently Gnome 40 has dropped all functionality for status icon/appindicators because they want to "grow out of the status tray" (sigh. I will never understand Gnome developers), so a shell extension is required to have the functionality back:
- [ubuntu/gnome-shell-extension-appindicator](https://github.com/ubuntu/gnome-shell-extension-appindicator) (officially from the Ubuntu team and recommendend by myself)
- [TopIcons](https://extensions.gnome.org/extension/495/topicons/) (recommended by Gnome)

I will also port the script from `GtkStatusIcon` to `libappindicator` for better support

**Comment by Baldomo** [comment]
Fixed in d2afe03, closing yhe issue
